src/main/java/spotify/api/impl/BrowseApiRetrofit.java

```diff
@@ -106,6 +140,34 @@ public CategoryFullPaging getCategories(Map<String, String> options) {
         }
     }
 
+    @Override
+    public Paging<CategoryFull> getCategoriesUnwrapped(Map<String, String> options) {
```

Comment:
Same as the comments at `getCategoryPlaylistsUnwrapped`

Reply:
The requested changes have been implemented.